web/docs/auth/email.md

```diff
@@ -167,27 +167,27 @@ Add the following to the `main.wasp` file:
 // 6. Define the routes
 route LoginRoute { path: "/login", to: LoginPage }
 page LoginPage {
-  component: import { Login } from "@client/pages/auth.jsx"
+  component: import { Login } from "@src/pages/auth.jsx"
```

Comment:
You can also remove `.jsx`, right? I started removing the extensions in the ext imports in wasp files. Not sure how we want to do it, we don't have to be uniform, but I found that to be the simplest.

Reply:
I didn't want to do that now as well, I guess it's fine if we keep some of the extensions?